docs/source/using/image-caching.adoc

```diff
@@ -4,39 +4,150 @@ include::variables.adoc[]
 :icons:
 :toc: macro
 :toc-title:
-:toclevels: 1
+:toclevels: 2
 
 toc::[]
 
 [[image-caching-overview]]
 == Overview
 
-To speed up provisioning of the OpenShift cluster and to minimize network traffic, the core OpenShift images can be cached on the host.
-This feature is considered experimental and needs to be explicitly enabled.
+To speed up the provisioning of the OpenShift cluster and to minimize network traffic, container images can be cached on the host.
+These images can then be imported into the running Docker daemon, either explicitly on request or implicitly during xref:../using/basic-usage.adoc#minishift-start-overview[`minishift start`].
+The following sections describe image caching and its configuration in more detail.
 
-[[image-chaching-configuration]]
-== Image Caching Configuration
+[TIP]
+====
+The format in which images are cached has changed with {project} version 1.10.0.
+Prior to 1.10.0 the images were stored as tar files.
+As of 1.10.0 images are stored in the link:https://github.com/opencontainers/image-spec/blob/master/spec.md[OCI image format].
+
+If you used image caching prior to {project} 1.10.0 your cache will need to be re-created.
+If you want to remove the obsolete pre 1.10.0 images, you can clear your cache via:
+----
+$ minishift delete --clear-cache
+----
+====
+
+[[explicit-image-caching]]
+== Explicit Image Caching
+
+{project} provides the `image` command together with its sub-commands to control the image caching behavior.
```

Comment:
Feel free to discard this suggestion (it's purely stylistic), but "to control the behavior of image caching" sounds better to me.

Reply:
ok. no strong feelings either way.